### PR TITLE
add vmui for vmselect component

### PR DESCRIPTION
**Pull request by TinySong**
fix: https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1413
Preliminary code implementation, please check out if there are any points that need to be optimized. thanks

### CODE REVIEW

File: app/vmselect/Makefile [hunk 1]
```diff
@@ -8,64 +8,64 @@ run-vmselect:
 	$(MAKE) run-via-docker
 
 vmselect:
-	APP_NAME=vmselect $(MAKE) app-local
+	APP_NAME=vmselect $(MAKE)vmui app-local
```
Comment (hagen1778) [inline app/vmselect/Makefile:11]: nit: `space` missed

File: deployment/docker/Makefile [hunk 2]
```diff
@@ -82,6 +87,7 @@ publish-via-docker: \
 		--push \
 		bin
 
+
```
Comment (hagen1778) [inline deployment/docker/Makefile:90]: nit: extra new line

File: deployment/docker/Makefile [hunk 3]
```diff
@@ -7,6 +7,11 @@ CERTS_IMAGE := alpine:3.14.0
 GO_BUILDER_IMAGE := golang:1.16.5
 BUILDER_IMAGE := local/builder:2.0.0-$(shell echo $(GO_BUILDER_IMAGE) | tr : _)
 BASE_IMAGE := local/base:1.1.3-$(shell echo $(ROOT_IMAGE) | tr : _)-$(shell echo $(CERTS_IMAGE) | tr : _)
+VMUI_VERSION=v0.1.0
+
+vmui:
+	cd app/vmselect/ui && \
+	wget  https://github.com/VictoriaMetrics/vmui/releases/download/${VMUI_VERSION}/static.zip -O /tmp/static.zip; unzip /tmp/static.zip -d /tmp; rm -rf ./*; mv /tmp/build/* ./
```
Comment (hagen1778) [inline deployment/docker/Makefile:14]: nit: extra `space` between `wget` and address.
Also, long command lines can be broke into multiple lines via `\` char.

Comment (TinySong) [reply deployment/docker/Makefile]: yeah, thanks you take the time to review the PR, fixed, pushed again.

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@TinySong , thanks for the contribution!

**Pull request merged**